packages/@tailwindcss-upgrade/src/utils/extract-static-plugins.test.ts

```diff
@@ -67,7 +90,87 @@ describe('findStaticPlugins', () => {
           plugins: [plugin1, 'plugin2', require('./plugin3')]
         }
     `),
-    ).toEqual(['./plugin1', 'plugin2', './plugin3'])
+    ).toEqual([
+      ['./plugin1', null],
+      ['plugin2', null],
+      ['./plugin3', null],
+    ])
+  })
+
+  test('can extract plugin options', () => {
+    expect(
+      findStaticPlugins(js`
+        import plugin1 from './plugin1'
+        import * as plugin2 from './plugin2'
```

Comment:
wait… does this work? I feel like this would produce `plugin2.default()` as a function right?

Reply:
@thecrypticace Ah yep you are right this should probably not work. I thought of the `loadModule` helper that will call `.default` if the object has it but this obviously does not apply here since we don't load from an ID.